mozci/push.py

```diff
@@ -78,12 +56,10 @@ def backedoutby(self):
         Returns:
             str or None: The commit revision which backs this push out (or None).
         """
-        global backouts
         backout_rev = self._hgmo.get('backedoutby')
         if not backout_rev:
             return None
 
-        backouts[backout_rev] = True
```

Comment:
This will make us perform more requests, not really so important though since we are going to switch to getting all backouts via ActiveData.

You might want to simplify this function since you're here, ```return self._hgmo.get('backedoutby') or None```

Reply:
It actually won't because the `HGMO` class uses an instance cache. So it should be the same number, though as you noted previously, more memory consumption.

Reply:
In this case, we have a HGMO instance for the push revision, but not for the backout one and we were adding the backout revision to the "backouts" cache.
After the change, we'll have to get automation relevance for the backout revision.

Reply:
Ah I see now, thanks! I can file an issue to add it back if you like, or if we'll be using AD soon then maybe not much point.